CHANGELOG.md

```diff
@@ -56,19 +56,19 @@
 - Bugfix: Fixed viewer list not closing after pressing escape key. (#3734)
 - Bugfix: Fixed links with no thumbnail having previous link's thumbnail. (#3720)
 - Bugfix: Fixed message only showing a maximum of one global FrankerFaceZ badge even if the user has multiple. (#3818)
-- Bugfix: Add icon in the CMake macOS bundle. (#3832)
-- Bugfix: Adopt popup windows in order to force floating behavior on some window managers. (#3836)
-- Bugfix: Fix split focusing being broken in certain circumstances when the "Show input when it's empty" setting was disabled. (#3838, #3860)
+- Bugfix: Added an icon in the CMake macOS bundle. (#3832)
+- Bugfix: Adopted popup windows in order to force floating behavior on some window managers. (#3836)
+- Bugfix: Fixed split focusing being broken in certain circumstances when the "Show input when it's empty" setting was disabled. (#3838, #3860)
 - Bugfix: Always refresh tab when a contained split's channel is set. (#3849)
-- Bugfix: Drop trailing whitespace from Twitch system messages. (#3888)
-- Bugfix: Fix crash related to logging IRC channels (#3918)
+- Bugfix: Dropped trailing whitespace from Twitch system messages. (#3888)
```

Comment:
I don't think this is right. To me this reads as we were dropping support for something. This changelog entry doesn't drop trailing whitespace from twitch system messages, instead Chatterino now drops it.

Reply:
Do you think `removed` reads better? Or would u like the Changelog itself changed to something more like:

```suggestion
- Bugfix: Fixed an issue where Anonymous gift messages would be larger than normal gift messages. (#3888)
```

Reply:
:+1: on the full rewrite